azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/error/ChatCompositeErrorCode.kt

```diff
@@ -5,7 +5,7 @@ package com.azure.android.communication.ui.chat.error
 
 import com.azure.android.core.util.ExpandableStringEnum
 
-internal class ErrorCode : ExpandableStringEnum<ErrorCode?>() {
+internal class ChatCompositeErrorCode : ExpandableStringEnum<ChatCompositeErrorCode?>() {
```

Comment:
You have ChatCompositeErrorCode exposed in the public ChatCompositeErrorEvent.
ChatCompositeErrorCode needs to be created in the model package in Java. See CallComposite for example.